rules-tests/CodeQuality/Rector/ClassMethod/OptionalParametersAfterRequiredRector/Fixture/update_params_order_of_static_method_call.php.inc

```diff
@@ -0,0 +1,15 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Rector\Tests\CodeQuality\Rector\ClassMethod\OptionalParametersAfterRequiredRector\Fixture;
+
+final class WebUtil 
+{   
+ 	  public static function radioList($name, $items = [], $selected, $class = "", $groupingCount = 0, $groupingClass = "", $required = false)
+    {
+               
+    }
+}
+
+WebUtil::radioList("inPersonOrVirtual", $inPersonOrVirtualItems, $selected, "", 1, "col-sm-3 margin-bottom-10")
```

Comment:
missing `;`

```suggestion
WebUtil::radioList("inPersonOrVirtual", $inPersonOrVirtualItems, $selected, "", 1, "col-sm-3 margin-bottom-10");
```

btw, what the expected result for it? could you update PR title and description for it ?

Reply:
Hey! this ir related to this discussion: https://github.com/rectorphp/rector/discussions/7397